src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -537,7 +587,7 @@ public static PoissonRegressionTrainer PoissonRegression(this RegressionCatalog.
         }
 
         /// <summary>
-        /// Predict a target using a linear multiclass classification model trained with the <see cref="LogisticRegressionMulticlassClassificationTrainer"/> trainer.
+        /// Predict a target using a linear multiclass classification model trained with the <see cref="LbfgsMaximumEntropyTrainer"/> trainer.
         /// </summary>
```

Comment:
Do you think it's worth it to add a few words saying that this is the multi class extension of logistic regression? (here and below)

Reply:
It will become
```csharp
/// Predict a target using a maximum entropy classification model trained with the L-BFGS method implemented in <see cref="LbfgsMaximumEntropyTrainer"/>.
```


---
In reply to: [267062024](https://github.com/dotnet/machinelearning/pull/2976#discussion_r267062024) [](ancestors = 267062024)